src/SignIn/SignInScreen.js

```diff
@@ -26,6 +39,55 @@ const styles = StyleSheet.create({
 });
 
 export default function SignInScreen({ navigation }) {
+  const [firstName, setFirstName] = useState('');
+  const [lastName, setLastName] = useState('');
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [number, setNumber] = useState('');
+  const [address, setAddress] = useState('');
+  const [hidePass, setHidePass] = useState(true);
+
+  const lastNameInput = useRef();
+  const emailInput = useRef();
+  const passwordInput = useRef();
+  const numberInput = useRef();
+  const addressInput = useRef();
+
+  const handleSignUp = () => {
+    base('Users').create([
+      {
+        fields: {
+          email,
+          password,
+          'first name': firstName,
+          'last name': lastName,
+          'phone number': number,
+          address,
+        },
+      },
+    ], (err, records) => {
+      if (err) {
+        console.error(err);
```

Comment:
Can you replace the console.log statements with an alert/etc that shows up on screen?